Review-08-Numerical-Integration-and-Differentiation.ipynb

```diff
@@ -275,7 +275,7 @@
     "\n",
     "(b) Like interpolation, quadrature using polynomials of large degree has the potential to introduce oscillations that may bear no relation to the data to be fit.\n",
     "\n",
-    "(c) Not sure."
+    "(c) Halving the subinterval reduces the error by a factor of about 1/16."
```

Comment:
Could you explain this one? I see that in section 8.3 halving the interval of integration $[a, b]$ decreases the error in the trapezoid rule by 1/8.  Is the idea that by halving the step size $h$, we halve the interval of integration and thus decrease the error over any single interval by 1/8 * 1/2 = 1/16?  If so, could you mention this in the answer.  I think that might help someone reading in the future.

Reply:
You are right, it should be 1/8,  not 1/16.